src/sv-pipeline/scripts/format_svtk_vcf_for_gatk.py

```diff
@@ -32,7 +32,7 @@ def _parse_bnd_ends(vcf_path: Text) -> Dict[Text, int]:
             columns = line.split('\t', 8)
             vid = columns[2]
             info = columns[7]
-            if 'SVTYPE=BND' not in info:
+            if 'SVTYPE=BND' not in info and 'SVTYPE=CTX' not in info and 'SVTYPE=CPX' not in info:
```

Comment:
Totally fine with three things. If there's some possibility later on of having a lot more things that will be treated like BNDs / have END2 tags, you may want to consider storing these svtype declarations in a tuple and doing something like:
`if not any(svtype_declaration in info for svtype_declaration in svtype_declarations):`